container/nginx/confs/proxy.tlsconf

```diff
@@ -50,6 +50,21 @@ server {
         return 302 https://strn.network;
     }
 
+    location ~ ^/(ipns|api)/ {
```

Comment:
I'd like to see even a super low cache 1m

Reply:
I think thats reasonable.  will make it a TODO